device/barefoot/x86_64-accton_wedge100bf_32x-r0/montara/buffers_defaults_t0.j2

```diff
@@ -63,21 +63,17 @@
     },
 {%- endmacro %}
 
-{%- macro generate_pg_profils(port_names) %}
```

Comment:
Why is this removed? 3-4 should use the pg_lossless profiles?

Reply:
There is the correct template in [buffers_config.j2](https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/buffers_config.j2#L160-L178). If we remove this statement from buffers_defaults_t0.j2, buffers_config.j2 won't import macro generate_pg_profils from buffers_defaults_t0.j2 and use the correct template of BUFFER_PG.